src/App.tsx

```diff
@@ -1,156 +1,98 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { TodoAppHeader } from './components/TodoAppHeader/TodoAppHeader';
+import { TodoAppContent } from './components/TodoAppContent/TodoAppContent';
+import { TodoAppFooter } from './components/TodoAppFooter/TodoAppFooter';
+import { Notifications } from './components/Notifications/Notifications';
+import { client } from './utils/fetchClient';
+import { Todo } from './types/Todo';
+import { FilterType } from './types/FilterType';
+import { ErrorType } from './types/ErrorType';
+
+const USER_ID = 10308;
+
+const prepareTodos = (todoList: Todo[] | null, filterType: FilterType) => (
```

Comment:
handle it when you call this fucntion
```suggestion
const prepareTodos = (todoList: Todo[], filterType: FilterType) => (
```